test/check-application

```diff
@@ -857,7 +857,45 @@ class TestApplication(testlib.MachineCase):
         b.wait_text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(3)", "")
         b.wait_text("#containers-containers tbody tr:contains('busybox:latest') > td:nth-child(4)", "")
 
-    def testCheckpointRestoreSystem(self):
+    @testlib.nondestructive
+    def testCheckpointRestoreCGroupsV2(self):
+        b = self.browser
+        m = self.machine
+
+        if not self.cgroupsV2():
+            return;
+
+        # Only Fedora works with CGroupsV2
+        if m.image in ["fedora-32", "fedora-33"]:
```

Comment:
FYI: Debian's systemd switched to cgroupsv2 a few days ago, so we'll soon get it there as well.